src/alert-prelude.c

```diff
@@ -676,15 +676,19 @@ static int JsonToAdditionalData(const char * key, json_t * value, idmef_alert_t
 static void PacketToDataProtoHTTP(const Packet *p, const PacketAlert *pa, idmef_alert_t *alert)
 {
     json_t *js;
+    json_error_t error;
+    JsonBuilder *jsb;
 
-    js = JsonHttpAddMetadata(p->flow, pa->tx_id);
+    jsb = jb_new_object();
```

Comment:
`jb_new_object` allocates memory, which is not freed in this function (or in any of the others). End with a call to `jb_free`.

Also need error checking.

Reply:
done

Reply:
>I think for 6 (master-6.0.x) we can accept this approach, but for the suricata 7 side (master) I would like to get rid of it. Then there are several options: 1) see if it fits our new output plugin support 2) make it a completely separate tool that post processes EVE.
Next version of IDMEF will be in JSON I think so all of this will be easy :)

>It seems that there aren't many users anyway. The only report we've had was on the debian package not building. But no actual users complaining yet.
No report because no bug :) ? I can't find the bug you are talking about. Most of users that use the prelude support are on Centos/Redhat/Fedora.